packages/flutter/test/widgets/form_test.dart

```diff
@@ -1457,4 +1457,38 @@ void main() {
 
     expect(focusNode2.hasFocus, isTrue);
   });
+
+  testWidgets('AutovalidateMode.always should validate on first build', (WidgetTester tester) async {
+    String errorText(String? value) => '$value/error';
+
+    await tester.pumpWidget(
+      MaterialApp(
+        theme: ThemeData(),
+        home: Center(
+          child: Form(
+            autovalidateMode: AutovalidateMode.always,
+            child: Material(
+              child: Column(
+                children: <Widget>[
+                  TextFormField(
+                    initialValue: 'foo',
+                    validator: errorText,
+                  ),
+                  TextFormField(
+                    initialValue: 'bar',
+                    validator: errorText,
+                  ),
+                ],
+              ),
+            ),
+          ),
+        )
+      )
+    );
+
+    await tester.pump();
```

Comment:
It looks like this still takes 2 frames due to the addPostFrameCallback above? Ideally we wouldn't even need this pump here, only pumpWidget.

Reply:
I believe it used to work, but since I moved the business logic from the `Form` to the `FormField`, it no longer occurs.